internal/plugin.go

```diff
@@ -342,6 +386,46 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 				policy.Manifests[i].ComplianceType = policy.ComplianceType
 			}
 		}
+
+		for _, plcsetInPlc := range policy.PolicySets {
+			if _, ok := plcsetToPlc[plcsetInPlc]; !ok {
+				newPlcset := types.PolicySetConfig{
+					Name:     plcsetInPlc,
+					Policies: []string{policy.Name},
```

Comment:
Optional: After thinking about this, we may not need to set the slice here since it's overwritten in the `for i := range p.PolicySets {` loop below. This would conserve a little memory.